Yosemite/Yosemite/PointOfSale/POSProduct.swift

```diff
@@ -0,0 +1,33 @@
+
+public struct POSProduct: POSItem {
```

Comment:
❓ `POSProduct`'s public use seems to be for SwiftUI previews in the app layer, maybe we can create a separate implementation of `POSItem` for previews within DEBUG and name it like `POSPreviewItem`/`POSItemForPreview` (I'm not good at naming things)? In that case, `POSProduct` can probably be internal within Yosemite.

Reply:
Sounds good, added to the other SwiftUI preview cases that use Yosemite as dependency on https://github.com/woocommerce/woocommerce-ios/issues/12917